handler/src/main/java/io/netty/handler/ssl/SniHandler.java

```diff
@@ -53,12 +54,12 @@
      * @param mapping the mapping of domain name to {@link SslContext}
      */
     @SuppressWarnings("unchecked")
-    public SniHandler(DomainNameMapping<? extends SslContext> mapping) {
+    public SniHandler(Mapping<? super String, ? extends SslContext> mapping) {
```

Comment:
@trustin also does `? super String` make any sense ? I mean String is final anyway.